iFixit/react-commerce
# Improve code sharing

I've worked with @andyg0808 to make it possible to share code via the monorepo, so that it would be easier to share and reuse pieces of this repo. This will help to accomplish #245 

## Changelog

- Migrate to pnPM
- Fixed a few missing peer dependecies according to pnPM
- Create `ui` and `tsconfig` packages: `ui` contains a dummy `TestButton` component that can be used to verify that Next.js community project is able to correctly consume these packages via the monorepo.
- Refactored `tsconfig.json`

## QA

1. Visit the Vercel PR preview (replace `vercel.app` with `cominor.com`)
2. Visit all parts product list 
3. Verify that everything works the same way as for the main branch
4. Import `TestButton` from nextjs community project and verify that it renders correctly
